### PR TITLE
[TfL] Remove contact information from report submission emails

### PR DESCRIPTION
**Pull request by davea**
For FD-2405

[skip changelog]

### CODE REVIEW

File: perllib/FixMyStreet/Cobrand/TfL.pm [hunk 1]
```diff
@@ -467,9 +467,7 @@ around 'munge_sendreport_params' => sub {
 
     $self->$orig($row, $h, $params);
 
-    if ($row->category eq "Countdown - not working") {
-        $params->{From} = [ $self->do_not_reply_email, $self->contact_name ];
-    }
+    $params->{From} = [ $self->do_not_reply_email, $self->contact_name ];
```
Comment (dracos) [inline perllib/FixMyStreet/Cobrand/TfL.pm:470]: always_use_reply_to is set for all UK cobrands, so I think the Reply-To header would also need removing, as that contains the user's email.

File: t/cobrand/tfl.t [hunk 2]
```diff
@@ -417,12 +412,14 @@ subtest "extra information included in email" => sub {
     $mech->clear_emails_ok;
     FixMyStreet::Script::Reports::send();
     my @email = $mech->get_email;
-    like $email[0]->header('From'), qr/"Joe Bloggs" <fms-report-$id-/;
+    is $email[1]->header('From'), '"TfL Streetcare" <fms-tfl-DO-NOT-REPLY@example.com>';
```
Comment (dracos) [inline t/cobrand/tfl.t:415]: This is now same check as line 424, should it still be checking 0, not 1?

**Review by dracos — CHANGES_REQUESTED** [review]
Couple of issues, have left note on FD as to whether this is actually wanted for all, or just the category that it was already working for and just that wasn't working properly.

File: t/cobrand/tfl.t (hunk 2 above)
Comment (davea) [reply t/cobrand/tfl.t:415]: Oops! Ta.

**Comment by davea** [comment]
Thanks for spotting those things. Client has confirmed contact data should be removed everywhere.

**Review by dracos — APPROVED** [review]
Thanks for checking!

**Comment by dracos** [comment]
Just going through old PRs, worth chasing this one up, so can get it live? Not sure why they didn't ever come back on it, but I assume they still want it

**Pull request merged**